modifierGui.py

```diff
@@ -0,0 +1,94 @@
+import re
+import json
+from alpacaModifier import AlpacaModifier
+
+# Can be played with.
+MIN_NUMBER_OF_ITEMS_IN_LIST = 4
+# There are quite a few questions of type: 'Sort the following list in alphabetical order.' We usually don't want to format those items.
+CHECK_IF_SORT_QUESTION = True
+# Check if there is a number in the instruction. Using numbered lists helps the LLM correctly answer queries that want a specific number of answers.
+CREATE_NUMBERED_LIST_IF_NUMBER_IN_INSTRUCTION = True
+NUM_MAP = {'one': 1, 'two': 2, 'three': 3, 'four': 4, 'five': 5, 'six': 6, 'seven': 7, 'eight': 8, 'nine': 9, 'ten': 10}
+
+SAVE_FILE = 'alpaca_new.json'
+
+class MyAlpacaModifier(AlpacaModifier):
+	def fill_values(self, instruction='', input='', old_output='', modified_output=''):
+		self.instruction = instruction
+		self.input = input
+		self.old_output = old_output
+		self.modified_output = modified_output
+
+	def save_prev_item(self):
+		if self.prev_item != None:
+			self.prev_item['instruction'] = self.instruction
+			self.prev_item['input'] = self.input
+			self.prev_item['output'] = self.modified_output
+			print(f'saving as:\n{self.prev_item}')
+			self.prev_item = None
+
+	def next_generator(self):
```

Comment:
This is the logic that iterates through all the entries and searches for a match. If it finds one, it will return the results. It is called on every 'next' click.